test/csuite/random_directio/main.c

```diff
@@ -1341,6 +1362,7 @@ main(int argc, char *argv[])
         }
 
         /* parent */
+        child_pid = pid;
```

Comment:
Below after line 1394 I think `child_pid` should be reset to zero.

Reply:
Fixed